plat/Config.uk

```diff
@@ -11,9 +11,33 @@ config UKPLAT_LCPU_MAXCOUNT
         default 1
 
 config HAVE_SMP
-        bool
-        default y if UKPLAT_LCPU_MAXCOUNT > 1
-        default n
+	bool
+	default y if UKPLAT_LCPU_MAXCOUNT > 1
+	default n
+
+menu "MP Configuration"
```

Comment:
Actually the `MP` comes from the common part (and this has to be removed here anyways). But with MP I wanted to state that it is just multiprocessor support. We could have big.LITTLE architectures that will also be used via this "SMP" API. The only thing that is wrong in that regard is the `CONFIG_HAVE_SMP` define. Otherwise, we never talk about symmetric.

Reply:
Hah okay fair enough. I just found that `MP` may not be obvious to the user, while SMP is something people are used to, so maybe `Multi-core Configuration` would be more clear. But since it goes away, that becomes a moot point :)